.travis.yml

```diff
@@ -116,15 +118,23 @@ matrix:
 
 before_install:
   - if [[ ! -z $TARGET_ARCH ]] ; then source ./scripts/travis/before_install.$TARGET_ARCH.sh ; fi
-  - if [[ $(uname -s) == 'Darwin' ]]; then sudo mdutil -i off /; fi;
+  - |
+    if [[ "${TRAVIS_OS_NAME}" == "linux" ]]; then
+      export JOBS=$((`nproc` + 1))
+    fi
+  - |
+    if [[ "${TRAVIS_OS_NAME}" == "osx" ]]; then
+      export JOBS=$((`sysctl -n hw.ncpu` + 1))
+      sudo mdutil -i off /
+    fi
```

Comment:
https://travis-ci.org/Project-OSRM/osrm-backend/jobs/177262393#L1117

Seems to work. I think this was only a problem on container machines not GCE.